lucene/CHANGES.txt

```diff
@@ -89,8 +89,8 @@ API Changes
 
 Improvements
 
-* LUCENE-9687: Hunspell support improvements: add SpellChecker API, support default encoding and
-  BREAK/FORBIDDENWORD/COMPOUNDRULE affix rules, improve stemming of all-caps words (Peter Gromov)
+* LUCENE-9687: Hunspell support improvements: add API for spell-checking and suggestions, support compound words,
+  fix various behavior differences between Java and C++ implementations, improve performance (Peter Gromov, Dawid Weiss)
```

Comment:
I don't think I deserve to be in the changes entry. :)

Reply:
I think you do, as you've dedicated quite a lot of your time to my and our own PRs. We could also change that to "mostly Peter Gromov, but also Dawid Weiss" if you like :)

Reply:
There's actually a convention for this: "Peter Gromov via Dawid Weiss".

Reply:
Doesn't matter.

Reply:
I thought "via" is for pure review activity, but you've made meaningful changes yourself which I didn't even plan. Feel free to change to "via" if you like :)